libbeat/docs/security/users.asciidoc

```diff
@@ -23,9 +23,16 @@ subset of the privileges needed by {beatname_uc} users. When possible, use the
 built-in roles to minimize the affect of future changes on your security
 strategy.
 
+Instead of using usernames and passwords, roles and privileges can be assigned to
+API keys to grant access to Elasticsearch resources. See <<beats-api-keys>> for
+more information
 
 [[privileges-to-setup-beats]]
-==== Grant privileges and roles needed for setup
+=== Grant privileges and roles needed for setup
```

Comment:
If it's alright, I'll probably move each role into its own file in `libbeat/docs/security/`.